### ISSUE TITLE
VST3 GUI on Linux just a grey box

### ISSUE THREAD

**Issue by dromer** [jpcima/Bass21#4]
When I open the VST3 GUI I just see this: https://i.imgur.com/MVCg20o.png

**Comment by jpcima** [comment]
What host is this? is the problem happening with all DAW software?

**Comment by dromer** [comment]
Sorry, this is using latest carla `2.3.0`.
Now also tried with Ardour 6.8 and there it works.

**Comment by jpcima** [comment]
The same here, carla 2.3.0 as dev host.
It's a nothing-special juce based editor.
Might the logs tab report anything relevant perhaps?

**Comment by dromer** [comment]
Indeed, the moment I load the plugin I see:  `JUCE Assertion failure in juce_AudioPluginInstance.cpp:203`

**Comment by jpcima** [comment]
From what I've collected, it's a bug of uncertain reproducibility, and it happens as result of a X11 BadWindow error.
I had reproducibility under Carla 2.3.1, but not under a git version of Carla.

Note that I had this problem occur, not only with this plugin, but with other plugins not based on Juce.
This bring me to suspicion that the source of this problem is possibly in Carla.
